.hooks/pre-push

```diff
@@ -14,7 +17,18 @@ npx eslint --quiet
 printf "\n⏳ npm run test\n"
 npm run test
 
-printf "\n⏳ composer test\n"
-vendor/bin/phpunit
+# Check the OS. Windows does not support the --parallel flag.
+if [ "$(uname)" = "Linux" ] || [ "$(uname)" = "Darwin" ]; then
+  # Use --parallel if macOS or Linux are detected.
+  printf "\n⏳ composer test -- --parallel\n"
+  vendor/bin/paratest &
+else
+  # If neither of those are detected, don't use --parallel.
+  printf "\n⏳ composer test\n"
+  vendor/bin/phpunit &
```

Comment:
could the trap/wait be moved into the paratest logic, leaving this to run in the foreground (no `&`)?

Reply:
trap/wait all removed in latest and paratest now runs in the foreground.